strongbox-commons/pom.xml

```diff
@@ -65,6 +65,13 @@
     </build>
 
     <dependencies>
+
+        <dependency>
```

Comment:
I can't remember the exact reason but we had to resolve something related to circular dependencies. I believe we still need that change strongly.